packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -399,6 +397,11 @@ export class StatsAnalyzer extends EventsScope {
 
     newMqa.intervalMetadata.peerReflexiveIP = this.statsResults.connectionType.local.ipAddress;
 
+    const numLogicalCores = CpuInfo.getNumLogicalCores();
+    if (numLogicalCores) {
+      newMqa.intervalMetadata.cpuInfo.numberOfCores = numLogicalCores;
+    }
```

Comment:
Up to you: I think this could be simplified to a single line `newMqa.intervalMetadata.cpuInfo.numberOfCores = CpuInfo.getNumLogicalCores() || 1;`